code/style.css

```diff
@@ -90,15 +192,33 @@ main {
 }
 
 .bubble {
-  background: #e5e9ff;
   font-weight: 600;
   font-size: 16px;
   line-height: 26px;
+  letter-spacing: 1.4;
   padding: 16px 24px;
-  color: #0026ff;
-  max-width: 40%;
+  margin-bottom: 16px;
+  text-transform: uppercases;
+  color: #303030;
+  max-width: 48%;
+  background: linear-gradient(-45deg, #edd7d0, #f6dce5, #d8e8ed, #e7f4f1);
+  background-size: 400% 400%;
+  animation: gradient 15s ease infinite;
+  height: 8S0vh;
+}
+@keyframes gradient {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
 }
 
+/*just bubbles, border radious not on the one corner makes it look like talk bubble.*/
```

Comment:
Making notes in CSS as well in addition to javascript is a good idea that I also should implement.